descriptors/mongodb/debezium-mongodb-1.9.0.Alpha1.json

```diff
@@ -1,23 +1,23 @@
 {
   "connector_type" : {
-    "id" : "debezium-mongodb-1.5.0.Final",
+    "id" : "debezium-mongodb-1.9.0.Alpha1",
     "kind" : "ConnectorType",
-    "href" : "/api/managed-services-api/v1/connector-types/debezium-mongodb-1.5.0.Final",
-    "name" : "debezium-mongodb",
-    "version" : "1.5.0.Final",
-    "title" : "Debezium MongoDB Connector",
+    "href" : "/api/connector_mgmt/v1/kafka_connector_types/debezium-mongodb-1.9.0.Alpha1",
+    "name" : "Debezium MongoDB Connector",
+    "version" : "1.9.0.Alpha1",
+    "channels" : [ "stable" ],
     "description" : null,
-    "icon_href" : "http://example.com/images/debezium-mongodb-1.5.0.Final.png",
-    "labels": ["source", "debezium", "mongodb", "1.5.0.Final"],
-    "channels" : ["stable"],
-    "json_schema" : {
+    "labels" : [ "source", "debezium", "mongodb", "1.9.0.Alpha1" ],
+    "capabilities" : [ "data_shape" ],
+    "icon_href" : "http://example.com/images/debezium-mongodb-1.9.0.Alpha1.png",
+    "schema" : {
       "title" : "Debezium MongoDB Connector",
       "required" : [ "mongodb.name" ],
       "type" : "object",
       "properties" : {
         "mongodb.name" : {
           "title" : "Namespace",
-          "description" : "Unique name that identifies the MongoDB replica set or cluster and all recorded offsets, and that is used as a prefix for all schemas and topics. Each distinct MongoDB installation should have a separate namespace and monitored by at most one Debezium connector.",
+          "description" : "Unique name that identifies the MongoDB replica set or cluster and all recorded offsets, andthat is used as a prefix for all schemas and topics. Each distinct MongoDB installation should have a separate namespace and monitored by at most one Debezium connector.",
```

Comment:
Typo.

Reply:
@gunnarmorling  this needs to be fixed upstream in debezium/debezium. We only consume the Field descriptions. Will do that upstream.

Reply:
see debezium/debezium#3211